### PR TITLE
DHFPROD-8093: Visual/Alignment issues from Ant Replacement

### PR DESCRIPTION
**Pull request by lucianaabc-ml**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/components/entities/mapping/entity-map-table/entity-map-table.module.scss [hunk 1]
```diff
@@ -100,8 +100,12 @@
 
 .renderContainer {
   padding-top: -5px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
 }
 
+
```
Comment (federicoRufrancosMarklogic) [inline marklogic-data-hub-central/ui/src/components/entities/mapping/entity-map-table/entity-map-table.module.scss:108]: remove this extra white space

File: marklogic-data-hub-central/ui/src/components/modeling/graph-view/side-panel/side-panel.module.scss [hunk 2]
```diff
@@ -41,12 +41,14 @@
         .sidePanelTabLabel {
             font-size: 18px;
         }
-        .tab-content {
+        div[class=tab-content] {
             display: none;
-        }
+        }
```
Comment (federicoRufrancosMarklogic) [inline marklogic-data-hub-central/ui/src/components/modeling/graph-view/side-panel/side-panel.module.scss:46]: adjust indentation

File: marklogic-data-hub-central/ui/src/components/modeling/graph-view/side-panel/side-panel.module.scss [hunk 3]
```diff
@@ -41,12 +41,14 @@
         .sidePanelTabLabel {
             font-size: 18px;
         }
-        .tab-content {
+        div[class=tab-content] {
             display: none;
-        }
+        } 
+
```
Comment (federicoRufrancosMarklogic) [inline marklogic-data-hub-central/ui/src/components/modeling/graph-view/side-panel/side-panel.module.scss:47]: remove extra spaces

File: marklogic-data-hub-central/ui/src/components/steps/steps.module.scss [hunk 4]
```diff
@@ -16,6 +16,11 @@
     }
 }
 
+    .title {
+        top: 1.5rem;
+    }
+
```
Comment (federicoRufrancosMarklogic) [inline marklogic-data-hub-central/ui/src/components/steps/steps.module.scss:22]: white spaces

**Review by federicoRufrancosMarklogic — CHANGES_REQUESTED** [review]

**Review by mrodrigu-marklogic — APPROVED** [review]

**Review by iveyMarklogic — APPROVED** [review]

**Review by federicoRufrancosMarklogic — APPROVED** [review]

**Pull request merged**